search-service/src/main/kotlin/com/egm/stellio/search/util/QueryUtils.kt

```diff
@@ -22,40 +22,28 @@ fun parseAndCheckQueryParams(
     val withAudit = hasValueInOptionsParam(
         Optional.ofNullable(queryParams.getFirst(QUERY_PARAM_OPTIONS)), OptionsParamValue.AUDIT
     )
-    val count = queryParams.getFirst(QUERY_PARAM_COUNT)?.toBoolean() ?: false
-    val ids = parseRequestParameter(queryParams.getFirst(QUERY_PARAM_ID)).map { it.toUri() }.toSet()
-    val types = parseAndExpandRequestParameter(queryParams.getFirst(QUERY_PARAM_TYPE), contextLink)
-    val temporalQuery = buildTemporalQuery(queryParams, contextLink)
-    val (offset, limit) = extractAndValidatePaginationParameters(
+    val temporalQuery = buildTemporalQuery(queryParams)
+    val queryParamsParse = parseAndCheckParams(
```

Comment:
`queryParamsParse` ? 🤔 

you could instead rename `queryParams` to `requestParams` in the method arguments and use `queryParams` here.